cpp/modules/luma.gl/core/src/model.h

```diff
@@ -72,18 +74,21 @@ class Model {
   wgpu::ShaderModule fsModule;
 
  private:
-  wgpu::Device _device;
-  std::shared_ptr<garrow::Table> _attributeTable;
-  std::shared_ptr<garrow::Table> _instancedAttributeTable;
-  std::vector<UniformDescriptor> _uniforms;
-
   void _initializeVertexState(utils::ComboVertexStateDescriptor* descriptor,
                               const std::shared_ptr<garrow::Schema>& attributeSchema,
                               const std::shared_ptr<garrow::Schema>& instancedAttributeSchema);
   auto _createBindGroupLayout(wgpu::Device device, const std::vector<UniformDescriptor>& uniforms)
       -> wgpu::BindGroupLayout;
 
   void _setVertexBuffers(wgpu::RenderPassEncoder pass);
+
+  wgpu::Device _device;
+  std::shared_ptr<garrow::Table> _attributeTable;
+  std::shared_ptr<garrow::Table> _instancedAttributeTable;
+  std::vector<UniformDescriptor> _uniformDescriptors;
+  // Retaining a reference to uniform buffers so they don't go out of scope before drawing
```

Comment:
Not sure we need this comment, we will need them again and again for each draw call...